src/odemis/driver/tlfw.py

```diff
@@ -186,24 +187,25 @@ def _flushInput(self):
 
             # Shouldn't be necessary, but just in case
             skipped = self._serial.read(1000) # More than 1000 chars => give up
-            logging.debug("Skipping input %s", skipped.encode('string_escape'))
```

Comment:
Is it important to use string_escape here?

Reply:
Yes, kind-of. At the very least, you *cannot* use `.decode("utf-8")`. You can only do that if you know that the data is UTF-8 encoded. So far, I've never seen any serial device using such encoding. In general, they are all limited to ascii (with some exception for some Japanese SEMs which also support some old japanese encodings).

Here, we want to display the received data. When the protocol is really some "random" bytes (eg, the tmcm), we need a dedicated function to show any useful. As this protocol is "human readeable", the shortcut was to use `encode('string_escape')`. However, that's not really supported in Python 3. As discussed in PR #1060, it should be possible to use `.decode("latin1", "backslashreplace")`.